<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeTicketWidget</class>
 <widget class="QWidget" name="ChangeTicketWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;航班改签&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget { 
    border: 1px solid #cccccc;
    border-radius: 4px;
    background-color: white;
}</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
border: none;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>出发地</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDepart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700; color:#000000;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_2 {
    border: 1px solid #cccccc;
    border-radius: 4px;
    background-color: white;
}</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
border: none;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>目的地</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblArrive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700; color:#000000;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_3 { 
    border: 1px solid #cccccc;
    border-radius: 4px;
    background-color: white;
}</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(153, 153, 153);
border: none;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>出发日期</string>
    </property>
   </widget>
   <widget class="QLabel" name="lineEditDate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700; color:#000000;&quot;&gt;选择日期&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>961</width>
     <height>51</height>
    </rect>
   </property>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>更多排序</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>耗时短-长</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>耗时长-短</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>价格高-低</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>到达早-晚</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>到达晚-早</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>起飞晚-早</string>
     </property>
    </item>
   </widget>
   <widget class="ClickableLabel" name="ontime">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;准点率高-低&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="ClickableLabel" name="depart">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;起飞时间早-晚&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="ClickableLabel" name="dijia">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;低价优先&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidgetNewFlight">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>971</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget { border: none; background-color: #F5F5F5; }；
setSpacing(2)；</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>150</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;原航班：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblOldFlight">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 2px solid #ff7d00; 
    border-radius: 4px;
    background-color: #ff7d00;
    color: white;
    font-size: 16px;
    height: 44px;
}

QPushButton:hover {
    border-color: #ff6600;
    background-color: #ff6600;
}

QPushButton:pressed {
    border-color: #e65c00;
    background-color: #e65c00;
}

QPushButton:disabled {
    border-color: #cccccc;
    background-color: #cccccc;
    color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>确认搜索</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>widget</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget_3</zorder>
  <zorder>widget_8</zorder>
  <zorder>listWidgetNewFlight</zorder>
  <zorder>label_4</zorder>
  <zorder>lblOldFlight</zorder>
  <zorder>btnSearch</zorder>
  <zorder>calendarWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickablelabel.h</header>
   <slots>
    <signal>clicked()</signal>
    <slot>on_switchLabel_clicked()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
